src/user-guide/writing-workflows/configuration.rst

```diff
@@ -65,10 +65,9 @@ Cylc provides syntax plugins for the following editors:
 
 .. _Cylc.tmbundle: https://github.com/cylc/Cylc.tmbundle
 .. _vscode-cylc: https://marketplace.visualstudio.com/items?itemName=cylc.vscode-cylc
-.. _language-cylc: https://atom.io/packages/language-cylc
 
 Atom
-   install the `language-cylc`_ extension.
+   install the "language-cylc" extension.
```

Comment:
atom.io is back online. Or can change this link to https://github.com/cylc/language-cylc if that's more likely to persist into the future

Reply:
Done.